hack/table-gen/table-gen.go

```diff
@@ -0,0 +1,243 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	REPattern                   = `(?s)<!--\s*TABLE-START\s* -->.*<!--\s*TABLE-END\s*-->`
+	SkipIdentifier              = `SKIP-ELEMENT`
+	RESkipPattern               = `<!--\s*` + SkipIdentifier + `\s*([^\s]+)\s*-->`
+	SkipWithAncestorsIdentifier = `SKIP-WITH-ANCESTORS`
+	RESkipWithAncestorsPattern  = `<!--\s*` + SkipWithAncestorsIdentifier + `\s*([^\s-]+)\s*-->`
+)
+
+var (
+	CRDFilename string
+	MDFilename  string
+	APIVersion  string
+	CRDKind     string
+)
+
+func main() {
+	flag.StringVar(&CRDFilename, "crd-filename", "../../installation/resources/crds/telemetry/tracepipelines.crd.yaml", "Full or relative path to the .yaml file containing crd")
+	flag.StringVar(&MDFilename, "md-filename", "../../docs/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md", "Full or relative path to the .md file containing the file where we should insert table rows")
+	flag.Parse()
```

Comment:
Is there a reason for choosing the `tracepipelines.crd.yaml` and `telemetry-03-tracepipeline.md` as default values for the `CRDFilename` and `MDFilename`? I think they should be required flags and in case they are not provided, then the program should terminate with an error